### ISSUE TITLE
Create and export reports for particular date ranges

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#8760]
> What application are you using?

Right-of-Way (ROW) Portal

> Describe the problem.

Trying to create reports based on fiscal years. We'll need this for budgeting. Primarily, we'll need to export reports on certain date ranges. The current Knack doesn't appear to have a date range option, only a created "before" or created "after" date. Not a created "between" these dates. 

> Describe the outcome you'd like to see when this feature is implemented.

We'll be able to pull reports for fiscal years and use it in our ATD revenue estimations. 

> Describe any workarounds you currently have in place or alternative solutions you've considered.

We manually just exported the CSV and manually filter, but the CSV doesn't have creation date. 

> Is there anything else we should know?

ASAP for budgeting :) 

> Requested By
Darren U.

Request ID: DTS22-103690

**Comment by ChrispinP** [comment]
Fiscal data is now updated for all TCP Project and Submission records. I added FY21 and FY22 filters to the Resubmittal Chart.

![image](https://user-images.githubusercontent.com/67915167/160495136-c2a8f7a6-732d-40d6-9d72-eb79627625df.png)

Can update reporting filters for any other reports that need them. Need to inquire if YTD metrics are still valuable or if both YTD and FY data should be present.

Advanced Search pages were updated to search in between two dates and by Resubmittal
